peachjam_search/models.py

```diff
@@ -149,15 +149,23 @@ def find_new_hits(self):
             else:
                 params[key] = values
         params["search"] = self.q
-        params["created_at__gte"] = self.last_alerted_at.isoformat()
 
         engine = SearchEngine()
         form = SearchForm(params)
         form.is_valid()
         form.configure_engine(engine)
         es_response = engine.execute()
+        hits = es_response.hits[:20]
```

Comment:
If we want the first two pages then you'll have to override the default page size on the search engine object before executing the search.

Reply:
ah right. changed